### DISCUSSION TITLE
2024新春贺词

### DISCUSSION THREAD

**Discussion by PythonSmall-Q** [XMOJ-Script-dev/XMOJ-Script#435] [General]
亲爱的小明的OJ增强脚本用户：

在这辞旧迎新的时刻，我谨代表开发组共青团组织、少先队组织和开发者们向大家致以最诚挚的新年祝福！

过去的一年，是开发组克难奋进、开拓创新的一年。在全球疫情的严峻挑战下，我们坚持以人民为中心，积极响应国家号召，努力为各位提供优质的软件产品和服务。我们不断提升自身的技术能力和专业素养，推动开发组的转型升级和创新发展。我们积极参与各项活动，践行社会责任，展现了开发组的良好形象和风采。我们团结协作，互相学习，共同成长，营造了开发组的和谐氛围和文化。

我们的成功也少不了各位用户的支持和帮助！在此，我要向大家表示衷心的感谢和崇高的敬意！

新的一年，是开发组迎难而上、奋发有为的一年。2024 年是中国共产党成立 103 周年，中国共产主义青年团成立104周年，也是实现第一个百年奋斗目标的关键之年。在新的历史起点上，我们要继续贯彻落实习近平总书记的重要指示精神，坚持以新发展理念为指导，紧跟时代潮流，把握发展机遇，充分发挥开发组和用户心连心的优势和特色，为建设社会主义现代化强国贡献力量。

2024 年是龙年，龙是中华民族的象征，寓意着吉祥、幸福、力量和希望。让我们以龙的精神，迎接新的挑战，创造新的辉煌！

最后，祝愿大家在新的一年里，身体健康，工作顺利，家庭幸福，万事如意！祝愿我们伟大的祖国繁荣昌盛，国泰民安！

小明的OJ增强脚本开发组
2024 年 2 月 4 日

https://www.xmoj.tech/discuss3/thread.php?tid=275

**Comment by PythonSmall-Q** [comment]
@langningchen @boomzero @zhouyiqing0304 解释一下：团组织是因为我是团员，少先队组织是因为有人可能是少先队的，开发者考虑到可能有人退队，没有入团

**Comment by langningchen** [comment]
@PythonSmall-Q 这是你自己写的，还是ChatGPT写的？

**Reply by PythonSmall-Q** [reply]
别小看我的文笔好吧，我好歹是道法30分的人

**Comment by PythonSmall-Q** [comment]
我写的，我没有chatgpt

**Comment by langningchen** [comment]
@PythonSmall-Q 你真厉害！
